cmd/main.go

```diff
@@ -60,7 +60,7 @@ func processMessagesWithReconnect(app config.AppConfig, interrupt chan os.Signal
 		c, err := prime.DialWebSocket(context.Background(), app)
 
 		if err != nil {
-			log.Error(err)
+			log.Errorf("error connecting to websocket: %w", err)
```

Comment:
Not sure what the behavior is if you wrap a log.Errorf. %v is usually for log statements.